src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -27,18 +29,19 @@
      */
     public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_STATEMENT, PREFIX_DESCRIPTION, PREFIX_TAG);
+                ArgumentTokenizer.tokenize(args, PREFIX_STATEMENT, PREFIX_DESCRIPTION, PREFIX_SOLUTION, PREFIX_TAG);
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_STATEMENT, PREFIX_DESCRIPTION)
+        if (!arePrefixesPresent(argMultimap, PREFIX_STATEMENT, PREFIX_DESCRIPTION, PREFIX_SOLUTION)
```

Comment:
An issue can have any number of solutions and description (mentioned as remark in UG), so it's possible for the user to add without s/ and d/. I guess this PREFIX_DESCRIPTION and PREFIX_SOLUTION can be removed.

Reply:
For solution i think we should have any number (including 0), i will fix that. But for issue description i am not very sure since description is related to the issue not to the solution, therefore, i think maybe we have one description for every issue?

Reply:
I think one description for one issue. If have more description, we need to think how to display it in UI.